.buildkite/run-cpu-test.sh

```diff
@@ -38,6 +38,8 @@ function cpu_tests() {
     set -e
     pip install -r vllm/requirements/test.txt
     pip install -r vllm/requirements/cpu.txt
+    pytest -v -s tests/kernels/test_cache.py -m cpu_model
+    pytest -v -s tests/kernels/test_mla_decode_cpu.py -m cpu_model
```

Comment:
`pytest -v -s tests/kernels -m cpu_model` doesn't work due to Triton imports (there are probably other issues as well). We can have a separate PR to make `pytest -v -s tests/kernels -m cpu_model` work (as well as improve test coverage for CPU kernels)